doc/api/net.md

```diff
@@ -226,6 +226,11 @@ added: v0.1.90
 Begin accepting connections on the specified `port` and `hostname`. If the
 `hostname` is omitted, the server will accept connections on any IPv6 address
 (`::`) when IPv6 is available, or any IPv4 address (`0.0.0.0`) otherwise.
+
+*Note*: Depending on the operating system, listening to an IPv6 address such
+as `'::'` may cause the `net.Server` to listen on an equivalent IPv4 address
```

Comment:
2. It seems NO problem with normal IPv6 addresses.  At least i confirmed it at Mac which does not attempt to listen equivalent IPv4 address for normal IPv6 address. (On contrast, At same machine, for `::`, it does).

Test result:
```
node -e 'net.createServer().listen({host:"fe80::82e6:50ff:fe16:4a2%en0", port:5555})'
```
Then run 
```
netstat -an | grep 5555
```

On Mac OS X EI Capitan 10.11.6
```
tcp6       0      0  fe80::82e6:50ff:.5555  *.*                    LISTEN     
```

On Ubuntu 16.04, unfortunately Node.JS 7.5.0 can not listen IPv6 address for some reason i don't know.
About windows, i will tested later.